docs/source/14_contribution/02_developer_contributor_guidelines.md

```diff
@@ -139,74 +139,48 @@ def count_truthy(elements: List[Any]) -> int:
 ```
 
 Ensure that your PR builds cleanly before you submit it, by running the CI/CD checks locally, as follows:
-
-To run E2E tests you need to install the test requirements which includes `behave`.
-We also use [pre-commit](https://pre-commit.com) hooks for the repository to run the checks automatically.
+* `make lint`: PEP-8 Standards (`pylint`, `flake8`)
+* `make test`: unit tests, 100% coverage (`pytest`, `pytest-cov`)
+* `make e2e-tests`: end-to-end tests (`behave`)
```

Comment:
I condensed all of the following section into these 3 bullets points to make it easier to read.